### PR TITLE
Add contributor guides

### PR DESCRIPTION
**Pull request by krkshitij**

### CODE REVIEW

File: docs/Charting-Concepts/GaugeChart.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+  - *GaugeValueFormat.Fraction*: Renders the chart value as a fraction.
+  - *GaugeValueFormat.Percentage*: Renders the chart value as a percentage. This is the default format.
+
+  Note: If the min value is non-zero and no formatter function is provided, the chart value will be rendered as a number.
+
+- **variant**: This prop determines the presentation style of the gauge chart. Set it to one of the following options:
+  - *GaugeChartVariant.SingleSegment*: Displays and announces the segment sizes as percentages.
+  - *GaugeChartVariant.MultipleSegments*: Displays and announces the segment sizes as ranges. This is the default variant.
+
+For more details, see [Fluent UI React Charting Examples - Gauge Chart (windows.net)](https://fluentuipr.z22.web.core.windows.net/heads/master/react-charting/demo/index.html#/examples/GaugeChart)
```
Comment (AtishayMsft) [inline docs/Charting-Concepts/GaugeChart.md:115]: update this to the fluent doc site page

File: docs/Charting-Concepts/HeatMapChart.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+- **xAxisNumberFormatString**: Use this prop to specify a custom string format for formatting numeric x-axis labels. For more information, refer to [d3-format | D3 by Observable (d3js.org)](https://d3js.org/d3-format#locale_format)
+
+- **yAxisNumberFormatString**: Use this prop to specify a custom string format for formatting numeric y-axis labels. For more information, refer to [d3-format | D3 by Observable (d3js.org)](https://d3js.org/d3-format#locale_format)
+
+- **xAxisStringFormatter**: Use this prop to provide a custom function for formatting string-based x-axis labels.
+
+- **yAxisStringFormatter**: Use this prop to provide a custom function for formatting string-based y-axis labels.
+
+For more details, see [Fluent UI React Charting Examples - Heat Map Chart (windows.net)](https://fluentuipr.z22.web.core.windows.net/heads/master/react-charting/demo/index.html#/examples/HeatMapChart)
```
Comment (AtishayMsft) [inline docs/Charting-Concepts/HeatMapChart.md:68]: same here

**Review by AtishayMsft — DISMISSED** [review]

File: docs/Charting-Concepts/GaugeChart.md (hunk 1 above)
Comment (krkshitij) [reply docs/Charting-Concepts/GaugeChart.md]: done

File: docs/Charting-Concepts/HeatMapChart.md (hunk 2 above)
Comment (krkshitij) [reply docs/Charting-Concepts/HeatMapChart.md]: done

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**